src/app/app.component.css

```diff
@@ -90,16 +90,19 @@
 }
 
 .side-result-list {
-  width: 450px;
+  width: 0;
   height: 100%;
   position: absolute;
   right: 0;
-  display: none;
+  visibility: hidden;
   background-color: white;
 }
 
 .side-result-list-open {
   display: block;
+  width: 450px;   
+  visibility: visible;
+  overflow: hidden;
 }
```

Comment:
I don't understand the aim of setting the width to `0` and `visibility: hidden` at the same time
And why not just add the `overflow: hidden` to `side-result-list` class

Reply:
visibility: hidden, hides the table but I still see the inkBar, so I forced the width to 0

Reply:
`overflow: hidden;` moved to `side-result-list`